elisp/guix-devel.el

```diff
@@ -338,6 +338,46 @@ Each rule should have a form (SYMBOL VALUE).  See `put' for details."
   (add-before 'guix-devel-indent-modify-phases-keyword-2))
 
 
+;;; String inside sexp
+
+(require 'edit-indirect nil t)
+
+(defconst guix-devel-code-block-syntax
+  '("description" "synopsis"))
```

Comment:
Please use `defvar` here: I think `defconst` should be used only when
there is a strong understanding that a variable shouldn't be modified by
users.  And since most of the variables may be modified by users without
making much harm, I almost always prefer `defvar`s.

Actually, I don't like that the regexp is recalculated every time the
block-edit command is called, so I would rather replace this variable
with a real regexp:

```elisp
(defvar guix-devel-code-block-regexp
  (rx (syntax open-parenthesis)
      (or "description" "synopsis")
      not-wordchar)
  "Regexp used by '\\[guix-devel-code-block-edit]'.")
```